pkg/console/console.go

```diff
@@ -0,0 +1,109 @@
+package console
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	v1 "github.com/openshift/api/config/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Builder provides a struct for console object from the cluster and a console definition.
+type Builder struct {
+	// console definition, used to create the pod object.
+	Definition *v1.Console
+	// Created console object.
+	Object *v1.Console
+	// api client to interact with the cluster.
+	apiClient *clients.Settings
+}
+
+// AdditionalOptions additional options for console object.
+type AdditionalOptions func(builder *Builder) (*Builder, error)
```

Comment:
Don't see where it's used here.

Reply:
finally it does not, I'll remove it on this stage and if it will be required we'll add it back later